packages/camera/camera_platform_interface/lib/src/types/video_capture_options.dart

```diff
@@ -43,6 +44,17 @@ class VideoCaptureOptions {
   /// Should only be set if a streamCallback is also present.
   final CameraImageStreamOptions? streamOptions;
 
+  /// When `true`, configures the recording to be a persistent recording.
+  ///
+  /// A persistent recording will only be stopped by explicitly calling [CameraController.stopVideoRecording]
+  /// and will ignore events that would normally cause recording to stop, such as lifecycle events.
+  ///
+  /// On Android, you must set this parameter to `true`
+  /// to avoid cancelling any active recording when calling [CameraController.setDescription].
```

Comment:
Do the other platforms not cancel the active recording when calling `CameraController.setDescription`? If so, would this not be better to default to true to keep platform consistency?